scripts/amoebagame.js

```diff
@@ -1,40 +1,70 @@
+        if (sys.auth(src) > 0 && ["/", "!"].indexOf(message.charAt(0)) > -1) {
+            sys.stopEvent();
+            sys.sendHtmlAll(messagePrefix + message, id);
+            return false;
+        }
+        if (role !== poke.length) { // poke.length - 1 if no battle with Arceus
+            sys.sendHtmlAll(messagePrefix + (role !== poke.length - 1 ? poke[role].cry : message), id);
+            if (battles[role] && battles[role] !== src && sys.isInChannel(battles[role], id)) {
+                var winner = sys.rand(0, 2) > 0 ? battles[role] : src,
+                    loser = winner === src ? battles[role] : src;
+                if (role < poke.length - 1) {
+                    SESSION.users(winner).amoeba.role++;
+                    if (SESSION.users(loser).amoeba.role > 0) {
+                        SESSION.users(loser).amoeba.role--;
+                    }
+                } else {
+                    SESSION.users(loser).amoeba.role = 0;
+                }
+                if (role === 0) {
+                    sys.sendHtmlAll("<hr><center>" + /*pokeIcon(role) +*/ "<b>" + htmlEscape(sys.name(battles[role])) + "</b> and " + /*pokeIcon(role) +*/ "<b>" + htmlEscape(sys.name(src)) + "</b> engaged in a fierce battle!</center><br /><b><font color='#0000ff'>" + htmlEscape(sys.name(winner)) + "</font></b> won and evolves into " + /*pokeIcon(SESSION.users(winner).amoeba.role) +*/ poke[SESSION.users(winner).amoeba.role].species + " while <b><font color='#ff0000'>" + htmlEscape(sys.name(loser)) + "</font></b> lost and remains a " + /*pokeIcon(SESSION.users(loser).amoeba.role) +*/ poke[SESSION.users(loser).amoeba.role].species + ".<hr>", id);
+                } else if (role > 0 && role < 5) {
+                    sys.sendHtmlAll("<hr><center>" + /*pokeIcon(role) +*/ "<b>" + htmlEscape(sys.name(battles[role])) + "</b> and " + /*pokeIcon(role) +*/ "<b>" + htmlEscape(sys.name(src)) + "</b> engaged in a fierce battle!</center><br /><b><font color='#0000ff'>" + htmlEscape(sys.name(winner)) + "</font></b> won and evolves into " + /*pokeIcon(SESSION.users(winner).amoeba.role) +*/ poke[SESSION.users(winner).amoeba.role].species + " while <b><font color='#ff0000'>" + htmlEscape(sys.name(loser)) + "</font></b> lost and unevolved into " + /*pokeIcon(SESSION.users(loser).amoeba.role) +*/ poke[SESSION.users(loser).amoeba.role].species + ".<hr>", id);
+                } else {
+                    sys.sendHtmlAll("<hr><center>" + /*pokeIcon(role) +*/ "<b>" + htmlEscape(sys.name(battles[role])) + "</b> and " + /*pokeIcon(role) +*/ "<b>" + htmlEscape(sys.name(src)) + "</b> engaged in a fierce battle of the Gods!</center><br /><b><font color='#0000ff'>" + htmlEscape(sys.name(winner)) + "</font></b> defeated <b><font color='#ff0000'>" + htmlEscape(sys.name(loser)) + "</font></b> and mercilessly reset their evolution back to " + /*pokeIcon(SESSION.users(loser).amoeba.role) +*/ poke[SESSION.users(loser).amoeba.role].species + ".<hr>", id);

```

Comment:
Sulcuta suggested it. Thought it sounded like a good idea. :x So want me to remove Arceus battle too?